install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -234,6 +234,12 @@ SCCs], is the preferred solution. A custom SCC can be created such that minimum
 and maximum group IDs are defined, group ID range checking is enforced, and a
 group ID of 5555 is allowed.
 
+[NOTE]
+====
+In order to use an SCC, you must first add it to the
+xref:../../admin_guide/manage_scc.adoc#add-scc-to-user-group-project[appropriate service account].
+====
```

Comment:
default is usually the appropriate one unless they've specified a name on the pod spec